Exercises/Java Scrip/Ex_3/index.js

```diff
@@ -0,0 +1,19 @@
+let age = 33;
+let firstName = "Arash";
+let learn = true;
```

Comment:
let lern=true; Consider using more meaningful variable names. So the name would be understandable not only to you, but to developer that reading your code for first time.